cmd/kube-dns/app/options/options.go

```diff
@@ -18,37 +18,43 @@ limitations under the License.
 package options
 
 import (
-	"net/url"
-	"os"
-
 	"fmt"
 	_ "net/http/pprof"
+	"net/url"
+	"os"
 	"strings"
 
 	"github.com/spf13/pflag"
+	"k8s.io/kubernetes/pkg/dns/fed"
 	"k8s.io/kubernetes/pkg/util/validation"
 )
 
 type KubeDNSConfig struct {
 	ClusterDomain  string
 	KubeConfigFile string
 	KubeMasterURL  string
+
 	HealthzPort    int
 	DNSBindAddress string
 	DNSPort        int
-	// Federations maps federation names to their registered domain names.
```

Comment:
It duplicates the usage in the flags description below and increases the maintenance cost.